README.md

```diff
@@ -139,6 +139,8 @@ vader.Profile["nickName"] = "Lord Vader";
 var newVader = await vader.UpdateAsync();
```

Comment:
Change this to `await vader.UpdateAsync();`

Reply:
After this section, add a new section: **Get and set custom attributes**

Reply:
Still need to update line 117, and add a section before line 120:

```
### Get and set custom attributes
(some explanation)

vader.Profile["homeworld"] = "Tattooine";
```

The note on line 120 will follow this 👍